rskj-core/src/main/java/co/rsk/mine/gas/provider/StableMinGasPriceProvider.java

```diff
@@ -63,17 +101,41 @@ private long calculateMinGasPriceBasedOnBtcPrice(long btcValue) {
         return minStableGasPrice / btcValue;
     }
 
-    @Override
-    public Coin getMinGasPriceAsCoin() {
-        return Coin.valueOf(getMinGasPrice());
+    private synchronized Future<Long> fetchPriceAsync() {
+        Future<Long> curFuture = priceFuture.get();
```

Comment:
nitpick:

By the context, we clearly can see that this is current. Buut, I would avoid such abbreviation. `currentFuture` isn't a big name, and wouldn't be awful use it fully.